decentraland/components/presentational/__snapshots__/Button.test.js.snap

```diff
@@ -19,6 +19,29 @@ exports[`Button renders the component 1`] = `
     }
   }
 >
-  TITLE
+  ENABLED BUTTON
+</Button>
+`;
+
+exports[`Button renders the component disabled button 1`] = `
+<Button
+  activeOpacity={1}
+  containerStyle={
+    Object {
+      "backgroundColor": "gray",
```

Comment:
It's nice when reviewing PRs to have the snapshots in here to see that indeed it is gray in this case

Reply:
Yeah :+1:

Reply:
Resolving this